modules/apps/portal-security-sso/portal-security-sso-openid-connect-impl/src/main/java/com/liferay/portal/security/sso/openid/connect/internal/OpenIdConnectSessionRegistryImpl.java

```diff
@@ -27,12 +28,29 @@
 
 import javax.servlet.http.HttpSession;
 
+import org.osgi.service.component.annotations.Component;
+
 /**
  * @author Istvan Sajtos
  */
-public class OpenIdConnectUtil {
+@Component(
+	service = {
+		OpenIdConnectSessionRegistry.class,
+		OpenIdConnectSessionRegistryImpl.class
+	}
+)
+public class OpenIdConnectSessionRegistryImpl
```

Comment:
Hey @csierra . Do you think "Registry" is a good name for this class (and its interface) ?

Reply:
hey... I am not sure... 
searching across the codebase with `ag --multiline 'public.* set[^{]*HttpSession'` yields no results for similar interfaces, only `setHttpSession` methods. 
For what it looks like we use `Registry` for classes that get, and maybe register and unregister, but do not get session or request as param. 
Looking for all methods called `setSomething` that are not called `set*Request` (that would just set the request somewhere) but receive the request as param yields more results:
`ag --depth 50 'public.* void set((?!Request).)*\([^)]*HttpServletRequest'`

alas performing the same search with `HttpSession` only yields this class as a result... so I would probably go with `Helper` or `Customizer`

at any rate include this investigation if you pick any name :-)

Reply:
Thanks for that. *Helper seems like a better name. A bit more "lightweight" and it less "stateful" than *Registry. I'll leave this PR as is with my latest commits and squash in another to send to Brian. So that I can link our conversation here.